docs/src/main/tut/monoid.md

```diff
@@ -60,23 +60,9 @@ l.foldMap(i => i.toString)
 ```
 
 To use this
-with a function that produces a tuple, we can define a `Monoid` for a tuple 
+with a function that produces a tuple, cats also provides a `Monoid` for a tuple
```

Comment:
I used cats here because it is available through `cats.std`. and I didn't notice the sentence below. I will add the detail about the instance below as well.